preprocess-sqlite-Copy1.ipynb

```diff
@@ -0,0 +1,2952 @@
+{
```

Comment:
Do we need to keep that file ?

Reply:
No, I will keep it locally because it helps with the dev process :D